crates/extractor/src/schedule_ext.rs

```diff
@@ -1,17 +1,59 @@
 use super::*;
 
+use model::{Block, Schedule, TimeBlock};
+
 pub(crate) trait ScheduleExt {
-  fn from_block(block: &ElementRef) -> Self
+  fn from_selection(selection: &ElementRef) -> Result<Self>
   where
     Self: Sized;
 }
 
+fn term(ssid: Option<&str>) -> Option<String> {
+  match ssid {
+    Some("202305") => Some(String::from("Summer 2023")),
```

Comment:
Would be better to make this more general, like extract month/year from "yyyymm", then match "01" => Winter, "05" =>  Summer, "09" => Fall